README.md

```diff
@@ -348,6 +348,9 @@ for an `ops` field. `*Operations` needs to be used instead.
 * `Future` constructors now accept `Request` as their argument.
 * Methods `AppendPush` and `SetResponse` accepts response `Header` and data
   as their arguments.
+* Method `Err` removed because it provokes an invalid error checking. You need
+  to check an error from `Get`, `GetTyped` or `GetResponse` with an addition
```

Comment:
‘with an addition check’ -> ‘in addition you can check’ or ‘with an additional check of’

Reply:
Thank you, updated.